### PR TITLE
f-takeawaypay-activation@0.1.0 - Generate component

### PR DESCRIPTION
**Pull request by oleksiishevchukje**
Initial commit to create f-takeawaypay-activation component structure

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

### CODE REVIEW

File: packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+
+export default {
+    name: 'TakeawaypayActivation',
```
Comment (DamianMullins) [inline packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue]: You don't need to set this property as vue-cli will add it automatically for you based on the file name.

File: packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    components: {},
```
Comment (DamianMullins) [inline packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue]: NIT: Removed empty object property.

File: packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue (hunk 1 above)
Comment (oleksiishevchukje) [reply packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue]: Sure, will correct. Was just auto generated code

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const TakeawaypayActivation = require('../../test-utils/component-objects/f-takeawaypayActivation.component');
+const takeawaypayActivation = new TakeawaypayActivation
```
Comment (siggerzz) [inline packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Change to:

`const takeawaypayActivation = new TakeawaypayActivation();`

Sorry, this is a bug that's now been fixed :)

File: packages/components/organisms/f-takeawaypay-activation/test/accessibility/axe-accessibility.spec.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { getAccessibilityTestResults } from '../../../../../../test/utils/axe-helper';
+
+const TakeawaypayActivation = require('../../test-utils/component-objects/f-takeawaypayActivation.component');
+const takeawaypayActivation = new TakeawaypayActivation
```
Comment (siggerzz) [inline packages/components/organisms/f-takeawaypay-activation/test/accessibility/axe-accessibility.spec.js]: ditto

File: packages/components/organisms/f-takeawaypay-activation/test-utils/component-objects/f-takeawaypayActivation.component.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+module.exports = class TakeawaypayActivation extends Page {
+
+    get component () { return $(COMPONENT); }
+
+    open () {
+        super.openComponent('organims', 'takeawaypayActivation-component');
+    }
+
+    waitForTestComponent () {
```
Comment (siggerzz) [inline packages/components/organisms/f-takeawaypay-activation/test-utils/component-objects/f-takeawaypayActivation.component.js]: Change to `waitForComponent`

File: packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue (hunk 1 above)
Comment (siggerzz) [reply packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue]: Sorry about that, we noticed it the other day :)

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js (hunk 3 above)
Comment (SachaZvetelman) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Shouldn't it be `takeawayPayActivation` and `TakeawayPayActivation`?

Comment (siggerzz) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Yeah I think so. I'll need to create a PR to address this in the generator. I'm happy to go through and update affected components as part of this 👍🏼

Comment (siggerzz) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: On second thought, there may not be an issue with the generator, and the casing was just the result of the name used when Oleksii generated the component :)

Comment (oleksiishevchukje) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: I planned to update the naming actually, just need to not miss anything :)

Comment (oleksiishevchukje) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: @SachaZvetelman I tried to rename everything from TakeawaypayActivation to TakeawayPayActivation, but a failing build without a clear reason. Maybe something related to the naming conventions, not sure yet. Returned it back and will try to do that separately.

Comment (SachaZvetelman) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: @oleksiishevchukje, sure, but please make sure you do rename this so it follows our naming conventions. Thanks!

**Review by siggerzz — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**